apps/site/components/ApiDocs/StabilityIndex/index.module.css

```diff
@@ -0,0 +1,48 @@
+.stabilityIndex {
```

Comment:
Can you add sizes here too? Like on the screenshot from the nodes-website Slack channel?

Reply:
good point, it had slipped my mind